openlibrary/components/MergeUI.vue

```diff
@@ -105,7 +105,7 @@ export default {
                     }
                 } catch (e) {
                     this.mergeOutput = e.message;
-                    this.mergeStatus = this.isSuperLibrarian() ? DO_MERGE : REQUEST_MERGE;
+                    this.mergeStatus = this.isSuperLibrarian ? DO_MERGE : REQUEST_MERGE;
```

Comment:
This change may be unrelated to this PR, and was already merged into main as a part of #8259.